internal/grpc_opts/grpc_opts.go

```diff
@@ -1,4 +1,4 @@
-package grpc_opts
+package grpcopts
```

Comment:
No need to remove underscore because rule has been excluded, I think

Reply:
Removed